frontend/test/metabase/scenarios/models/models.cy.spec.js

```diff
@@ -246,6 +246,7 @@ describe("scenarios > models", () => {
 
       cy.icon("join_left_outer").click();
       cy.wait("@schema");
```

Comment:
Again, waiting for the related request doesn't seem to be enough any more! :(
There is a slight delay between the response coming back and the browser actually rendering the contents of that response.